crates/rpc/rpc/src/debug.rs

```diff
@@ -565,11 +574,9 @@ where
                         );
                         let (res, _, db) = inspect_and_return_db(db, env, &mut inspector)?;
 
-                        let frame = inspector.into_geth_builder().geth_prestate_traces(
-                            &res,
-                            prestate_config,
-                            db,
-                        )?;
+                        let geth_builder = inspector.into_geth_builder();
+                        let frame = geth_builder.geth_prestate_traces(&res, prestate_config, db)?;
+                        self.drop_in_task(geth_builder);
```

Comment:
this could spawn >200 drop tasks per request when tracing an entire block.
unsure about the sideeffects when the node is under heavy load